config/locales/en.yml

```diff
@@ -227,5 +258,5 @@ en:
     formats:
       default: "%a, %d %b %Y %H:%M:%S %z"
       long: "%B %d, %Y %H:%M"
-      short: "%d %b %H:%M"
+      short: "%d %b, %Y"
```

Comment:
Does “Y” has to be capitalized?

Reply:
Have you checked the date format of the USA?? Your one is not correct. https://en.wikipedia.org/wiki/Date_format_by_country

Reply:
they have MDY - month - day - year